### PR TITLE
Auto update

### PR DESCRIPTION
**Pull request by taylorsmart**
Toggle for auto update on charts

### CODE REVIEW

**Review by mmoren01 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -16,8 +16,8 @@
     "@material-ui/icons": "^4.11.2",
     "@nivo/bar": "^0.67.0",
     "@nivo/core": "^0.67.0",
-    "argon2": "^0.27.2",
     "@react-hook/window-size": "^3.0.7",
+    "argon2": "^0.27.2",
```
Comment (Ciaddie) [inline package.json:20]: was this installed twice or just moved?

File: db/index.js [hunk 2]
```diff
@@ -8,23 +8,30 @@ const pool = new Pool({
...
+const pool = new Pool({
+  host: '18.218.164.26',
+  user: 'ubuntu',
+  password: 'ubuntupass', //Adjust this to the password
+  database: 'nvision',
+})
 
-const {Client} = require('pg');
 
+//
```
Comment (Ciaddie) [inline db/index.js:20]: I am sure there is a great reason for this and I am also not familiar with postgres, but I am curious to know why you kept this comment

**Review by Ciaddie — APPROVED** [review]

File: db/index.js (hunk 2 above)
Comment (taylorsmart) [reply db/index.js:20]: Ease of the demo - it's actually bad for me to post this on git - so I'll remove it down the line.

File: package.json (hunk 1 above)
Comment (taylorsmart) [reply package.json:20]: I had to install it locally.  For some reason it wasn't recognized when I pulled in.

So just moved! :)

**Pull request merged**